scarb/src/bin/scarb/commands/cache_path.rs

```diff
@@ -0,0 +1,10 @@
+use anyhow::{Ok, Result};
+
+use scarb::core::Config;
+
+#[tracing::instrument(skip_all, level = "info")]
+pub fn run(config: &Config) -> Result<()> {
+    let path = config.dirs().cache_dir.path_unchecked();
+    println!("{path}");
```

Comment:
I've noticed one more thing, sorry for not mentioning this before. 

In Scarb we use our own UI abstraction for printing text to users (https://github.com/software-mansion/scarb/blob/main/scarb/src/ui/mod.rs). 
This also allows us to structure all messages to json if needed (with `--json` arg). 

Please print with `config.ui()` here (as below).

```suggestion
    config.ui().print(ValueMessage::new("path", &path));
```